inlong-common/src/main/java/org/apache/inlong/common/pojo/sort/dataflow/field/format/DateFormatInfo.java

```diff
@@ -50,11 +50,15 @@ public class DateFormatInfo implements BasicFormatInfo<Date> {
     @Nullable
     private final SimpleDateFormat simpleDateFormat;
 
+    @JsonIgnore
+    @Nullable
+    private final ThreadLocal<SimpleDateFormat> threadLocal;
```

Comment:
There is a SimpleDateFormat in every field object. 
It is better to use DateFormat tool class.

Reply:
Use FastDateFormat